### ISSUE TITLE
About Equation 6 in paper

### ISSUE THREAD

**Issue by llfzllfz** [mohamedac29/S2-FPN#3]
In paper，Equation 6 is Fscale = A ⊙F2 + A⊙F2, but with the text description, I think it should be Fscale = A⊙F1 + A⊙F2.

**Comment by mohamedac29** [comment]
@llfzllfz You are right. it is typo error in the manuscript. I have made some changes in the manuscript will upload the updated version soon. Thanks for the comments.
